bash.html.markdown

```diff
@@ -25,7 +25,21 @@ Nearly all examples below can be a part of a shell script or executed directly i
 # As you already figured, comments start with #. Shebang is also a comment.
 
 # Simple hello world example:
-echo Hello world!
+echo "Hello world!"
```

Comment:
Ah, must be the version difference. The thing is, Bash 3.2 is still the one bundled on Macs, and there's a significant developer population using Macs. Does it work in 4.3 if you escape the `!` with a backslash? As in

`echo "Hello world\!"`